poc_external_exporter/prometheus-additional.yaml

```diff
@@ -0,0 +1,3 @@
+- job_name: "test"
+  static_configs:
+  - targets: ["node.demo.do.prometheus.io"]
```

Comment:
If there is more then one target it's standard list, so can be?:

```yaml
- job_name: "test"
  static_configs:
  - targets:
    -  "node.demo.do.prometheus.io"
    -  "another.target.io"
```

Reply:
I think so https://prometheus.io/docs/prometheus/latest/configuration/configuration/#static_config